### ISSUE TITLE
odlmaple: Permission denied (publickey).

### ISSUE THREAD

**Issue by charlierproctor** [snlab/odl-summit-2016-tutorial-vm#4]
When running `vagrant up`, the build fails with

```
==> default: bash: line 18: popd: directory stack empty
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by charlierproctor** [comment]
Weirdly enough, if I then connect using `vagrant ssh` and directly run:

```
git clone git@github.com:snlab/odlmaple.git
```

everything works fine...

**Comment by charlierproctor** [comment]
Potentially related: https://github.com/mitchellh/vagrant/issues/1303.

Maybe the current box doesn't support SSH forwarding during provisioning

**Comment by charlierproctor** [comment]
Solved with 8a65c983d03d55fa2e547248db9ee1eaf4504aa4
